frontend/components/TableContainer/DataTable/InternalLinkCell/InternalLinkCell.tsx

```diff
@@ -3,13 +3,21 @@ import classnames from "classnames";
 import { noop } from "lodash";
 
 import Icon from "components/Icon";
+import { IconNames } from "components/icons";
 
 const baseClass = "internal-link-cell";
 
 interface IInternalLinkCellProps {
   value: string;
   onClick?: () => void;
   className?: string;
+  /** iconName is the name of the icon that will be dislayed to the right
+   * of the text.
+   *
+   * NOTE: This component has not been tested with all icons. Callers should
+   * ensure that the specified icon displays properly. Some issue were observed
+   * with icon clipping, sizing etc. */
+  iconName?: IconNames;
```

Comment:
AFAICS, this component is only used by the MDM solutions [table](https://github.com/fleetdm/fleet/blob/2a194af6ed0f64dfe15fc31dee7e639357e2123f/frontend/pages/DashboardPage/cards/MDM/MDMSolutionsTableConfig.tsx#L49-L50) and we'll just omit the iconName prop for that usage. We could potentially remove this component altogether and style directly on the table, but I figured we may as well keep it since it does provide some utility as far as styling the text with on hover.

Reply:
I know I've seen this arrow somewhere before...

host details > policies (which is added using a css pseudo element `::after` and looks horrible when using keyboard to access the table

<img width="886" alt="Screenshot 2025-01-08 at 3 54 00 PM" src="https://github.com/user-attachments/assets/6567c2dd-dd7f-46c0-b7a3-9c0112e7ceb4" />

Reply:
@RachelElysia, assuming we want to remove the arrows from the policies table too, should we convert this comment into a follow up ticket and assign to product to confirm?